language/bytecode-verifier/src/abstract_state.rs

```diff
@@ -22,6 +25,17 @@ pub struct TypedAbstractValue {
     pub value: AbstractValue,
 }
 
+impl TypedAbstractValue {
+    pub fn reference_to(&self, type_sig: &SignatureToken) -> bool {
```

Comment:
Nit: `reference_to` => `is_reference_to` to make it consistent with other helper functions?